include/zephyr/bluetooth/hci_types.h

```diff
@@ -2402,12 +2402,70 @@ struct bt_hci_cp_le_tx_test_v4_tx_power {
 	int8_t tx_power;
 } __packed;
 
+#define BT_HCI_OP_LE_CS_READ_LOCAL_SUPPORTED_CAPABILITIES BT_OP(BT_OGF_LE, 0x0089) /* 0x2089 */
+
+struct bt_hci_rp_le_read_local_supported_capabilities {
```

Comment:
rp?

Reply:
> rp?

response parameters, a structure naming convention used throughout this file

Reply:
ah ok. I missed that it was for the event. We've moved this back to draft btw